core/templates/dev/head/pages/exploration_editor/feedback_tab/FeedbackTab.js

```diff
@@ -173,7 +192,8 @@ oppia.controller('FeedbackTab', [
             if (!$scope.canEdit) {
               $scope.errorMessage = '';
             } else if (!$scope.isOpen) {
-              $scope.errorMessage = suggestionStatus === 'fixed' ?
+              $scope.errorMessage = (suggestionStatus === 'accepted' ||
+                suggestionStatus === 'fixed') ?
```

Comment:
fixed?

Reply:
'Fixed' is for backward compatibility for the old framework.